### DISCUSSION TITLE
Improvement Discussion: Extending the Menu for managing existing Websites

### DISCUSSION THREAD

**Discussion by Dbochman** [sindresorhus/Plash#134] [General]
Hello everyone,

One of Plash's great features is the ability to add multiple websites and easily switch between them through a static dropdown list. While this is handy, I noticed that while managing the list the menu can only accommodate a certain number of links. As a user who utilizes more links [than the current interface can currently display](https://github.com/Dbochman/images/blob/main/Screenshot%202023-07-11%20at%2015.25.00.png?raw=true), I find this limitation somewhat restrictive.

Given this, I propose that we discuss ways to enhance this feature to cater to users with numerous links. One straightforward option could be to make the list scrollable, which would allow for an unlimited number of links without redesigning the current display format. However, I believe it's worth considering other innovative designs to ensure the best user experience.

For instance, we might think about:

A dynamic grid view: This could provide a more visually appealing way to display numerous links, especially if users prefer seeing webpage thumbnails.
Folders or categories: This could help users to organize their links, thereby making navigation smoother and quicker.
I understand that such changes could have wide-ranging impacts on the codebase and user experience. Therefore, I am opening this discussion to get your thoughts, opinions, and perhaps other innovative ideas to enhance this feature.

Looking forward to hearing your valuable feedback.

Best,
Dylan

**Comment by sindresorhus** [comment]
I really appreciate the feedback. Because of lack of time on my part and also an urge to keep Plash simple, I would recommend submitting smaller quality-of-life improvement suggestions rather than large feature requests.

> One straightforward option could be to make the list scrollable,

It is already scrollable.

> Folders or categories

This may be something I do at some point, but probably not soon. Plash will need a rewrite before I'm able to even consider changes like that.
